src/data/WorkshopSchedule.js

```diff
@@ -1,10 +1,123 @@
-// const hothYrMnth = '2022-02-'; // replace with real date later
-// const utcOffset = '-08:00'; // PST
+const hothYrMnth = '2022-02-'; // replace with real date later
+const utcOffset = '-08:00'; // PST
 // time is a string of the form "HH:mm:ss"
-// function getDateObj(day, time) {
-// 	return new Date(`${hothYrMnth}${day}T${time}${utcOffset}`);
-// }
+function getDateObj(day, time) {
+	return new Date(`${hothYrMnth}${day}T${time}${utcOffset}`);
+}
 
-const workshopSchedule = [];
+const workshopSchedule = [
+	{
+		startTime: getDateObj('26', '16:00:00'),
+		events: [
+			{
+				name: 'Check in',
+				duration: '60 min',
+				location: 'Covel Grand Horizon',
+				description: `Bring your bruin card, daily symptoms monitoring, and
+                your negative Covid test result and check in with one of the hack 
+                officers at the front.`
+			}
+		]
+	},
+	{
+		startTime: getDateObj('26', '17:00:00'),
+		events: [
+			{
+				name: 'Opening Ceremony',
+				duration: '30 min',
+				location: 'Covel Grand Horizon/Zoom',
+				description: `Come to the opening ceremony to hear a rundown of the rules
+				before you begin hacking.`
+			}
+		]
+	},
+	{
+		startTime: getDateObj('26', '17:30:00'),
+		events: [
+			{
+				name: 'Team Formation',
+				duration: '30 min',
+				location: 'Covel Grand Horizon/Discord',
+				description: `Looking for a team? Find one to hack with during
+				our team building session!`
+			},
+			{
+				name: 'Hacking Begins',
+				duration: '',
+				location: 'Covel Grand Horizon/Discord',
+				description: `Happy hacking! We hope you build something amazing!`
+			}
+		]
+	},
+	{
+		startTime: getDateObj('26', '20:00:00'),
+		events: [
+			{
+				name: 'Lunch',
+				duration: '60 min',
+				location: 'Covel Grand Horizon Terrace',
+				description: `Coding is done best on a full stomach ;). 
+				Come one, come all—we feast!`
+			},
+			{
+				name: 'JEDI Allyship Space',
+				duration: '60 min',
+				location: 'Covel Grand Horizon',
+				description: `Let's talk about being allies in CS`
```

Comment:
yeah definitely. I already asked them tbh. I just wrote this so I had something there because I realized that I had NEGATIVE CLUES what this event was about LMAOOO. 😆